lib/admin-page.php

```diff
@@ -453,7 +453,8 @@ function classicpress_check_can_migrate() {
 	$theme = wp_get_theme();
 	if ( isset( $parameters['themes'] ) &&
 		in_array( $theme->stylesheet, (array) $parameters['themes'] ) ||
-		( is_child_theme() && in_array( $theme->parent()->stylesheet, (array) $parameters['themes'] ) )
+		( is_child_theme() && in_array( $theme->parent()->stylesheet, (array) $parameters['themes'] ) ) ||
+		version_compare( $theme->get( 'RequiresWP' ), '5.0' ) >= 0
```

Comment:
As written, this reads the same as when we have added a theme to our hardcoded list of incompatible themes in the API. I think this should be a separate check instead. This check could then indicate which of the possible situations is present:

- The theme actually declares compatibility with WP 4.9+ or earlier (all good)
- The theme declares compatibility with WP 5.0+ or later (bad)
- The theme does not declare a compatibility header (unknown)

This way we can tell the user exactly what is being checked.

Adding more checks will mean that the migration plugin output is a bit long... I actually think this is fine. Migration is a complicated process and there is a lot that can go wrong.

Reply:
So, are you proposing a new section be added to the check steps that outputs an analysis of the theme header and then the next section compares the theme to out known list of incompatibilities as per the API?

Or should we just make the output under the 'theme' heading a little more specific in terms fo telling the user if their theme is declaring a problem or if we are aware of a problem?